backend/tests/v5/e2e/routes/teamspaces/projects/models/common/tickets.test.js

```diff
@@ -1277,6 +1297,115 @@ const testUpdateManyTickets = () => {
 	});
 };
 
+const testGetTicketHistory = () => {
+	describe('Get ticket history', () => {
+		const { users, teamspace, project, con, fed } = generateBasicData();
+		const template = ServiceHelper.generateTemplate();
+
+		const conTicket = ServiceHelper.generateTicket(template);
+		const fedTicket = ServiceHelper.generateTicket(template);
+
+		const conTicketUpdate = ServiceHelper.generateTicket(template);
+		const fedTicketUpdate = ServiceHelper.generateTicket(template);
+
+		const ticketUpdate = { title: ServiceHelper.generateRandomString() };
+		const updateDate = new Date();
+
+		beforeAll(async () => {
+			await setupBasicData(users, teamspace, project, [con, fed],
+				[template]);
+			await ServiceHelper.db.createTicket(teamspace, project, con, conTicket);
+			await ServiceHelper.db.createTicket(teamspace, project, fed, fedTicket);
+			await ServiceHelper.db.createTicket(teamspace, project, con, conTicketUpdate);
+			await ServiceHelper.db.createTicket(teamspace, project, fed, fedTicketUpdate);
+
+			// Simulate ticket update and log entry for conTicket
+			await insertOne(teamspace, 'tickets.logs', {
+				_id: ServiceHelper.generateUUIDString(),
+				author: users.tsAdmin.user,
+				changes: { properties: { title: { from: conTicket.title, to: ticketUpdate.title } } },
+				timestamp: updateDate,
+				teamspace,
+				project: stringToUUID(project.id),
+				model: con._id,
+				ticket: stringToUUID(conTicketUpdate._id),
+			});
+
+			// Simulate ticket update and log entry for fedTicket
+			await insertOne(teamspace, 'tickets.logs', {
+				_id: ServiceHelper.generateUUID(),
+				author: users.tsAdmin.user,
+				changes: { properties: { title: { from: fedTicket.title, to: ticketUpdate.title } } },
+				timestamp: updateDate,
+				teamspace,
+				project: stringToUUID(project.id),
+				model: fed._id,
+				ticket: stringToUUID(fedTicketUpdate._id),
+			});
+		});
+
+		const generateTestData = (isFed) => {
+			const modelType = isFed ? 'federation' : 'container';
+			const modelWithTemplates = isFed ? fed : con;
+			const ticketId = isFed ? fedTicket._id : conTicket._id;
+			const updateTicketId = isFed ? fedTicketUpdate._id : conTicketUpdate._id;
+			const modelNotFound = isFed ? templates.federationNotFound : templates.containerNotFound;
+			const wrongTypeModel = isFed ? con : fed;
+
+			const getHistoryRoute = (
+				{ key = users.tsAdmin.apiKey,
+					projectId = project.id,
+					modelId = modelWithTemplates._id,
+					ticket = ticketId,
+				} = {}) => `/v5/teamspaces/${teamspace}/projects/${projectId}/${modelType}s/${modelId}/tickets/${ticket}/history${key ? `?key=${key}` : ''}`;
+
+			const updateHistory = [
```

Comment:
once the feedback is addressed it would be good if you make a date change and Id change (ie image property) to ensure the transofmations are happening. You can include all these changes in a single log